assets/js/googlesitekit/datastore/user/user-info.js

```diff
@@ -473,13 +478,14 @@ export const selectors = {
 	 * Gets the user input state.
 	 *
 	 * @since 1.20.0
+	 * @since n.e.x.t Simplified user input state object to a simple boolean.
```

Comment:
Same here, this selector is new but the doc here is as if the old selector was kept and modified.